cdc/sinkv2/txneventsink/txn_event_sink.go

```diff
@@ -36,4 +39,35 @@ type TxnEventSink interface {
 	WriteTxnEvents(txns ...*TxnEvent) error
 	// Close closes the sink.
 	Close() error
+	// It's possible to dispatch tasks into several workers.
+	WorkerCount() int
+}
+
+// Worker is associated with a goroutine.
+type Worker interface{}
+
+type txnEventSinkImpl struct {
+	input   chan<- *TxnEvent
```

Comment:
I recommend against using a `chan`. Probably a custom implementation of a queue-like data structure is better.